scIB/metrics.py

```diff
@@ -351,22 +381,23 @@ def get_isolated_labels(adata, label_key, batch_key, cluster_key, n, verbose):
     return labels
 
 
-def score_isolated_label(adata, label_key, cluster_key, label, cluster=True, verbose=False, **kwargs):
+def score_isolated_label(
+        adata,
+        label_key,
+        label,
+        embed,
+        cluster=True,
+        cluster_key='iso_cluster',
+        verbose=False
+):
     """
     compute label score for a single label
     params:
         cluster: if True, use clustering approach, otherwise use silhouette score approach
+        kwargs:
```

Comment:
this was removed, so no longer need kwargs. Could you complete the function description here though?